src/Materials/standardMaterial.ts

```diff
@@ -193,11 +197,25 @@ export class StandardMaterialDefines extends MaterialDefines
     public IS_REFRACTION_LINEAR = false;
     public EXPOSURE = false;
 
-    constructor() {
+    protected _keysFromPlugins?: string[];

```

Comment:
Same comment for `_keysFromPlugins`, ctor and `rebuild` function than for the PBR base material.